source/game/map/Map.gd

```diff
@@ -401,15 +402,20 @@ func add_contruction(origin: Tile, data: ConstructionData) -> void:
 			valid_construction_sites[neighbor.position] = neighbor
 
 
+
+
 func _add_connection(tile: Tile) -> Connector:
-	rails_overlay.set_cellv(tile.position, 0)
-	rails_overlay.update_bitmask_area(tile.position)
+	_add_rail(tile)
 
 	var connector = Connector.new(tile)
 	connectors[tile.position] = connector
 
 	return connector
 
+func _add_rail(tile: Tile) -> void:
+	rails_overlay.set_cellv(tile.position, 0)
```

Comment:
Use constant instead of arbitrary values whenever possible

Reply:
changes pushed.